src/app/(admin)/admin/_utils/transformSpecialistFormData.js

```diff
@@ -6,15 +6,16 @@ const transformAddresses = placesArray =>
     district: { connect: { id: place.district } },
   }));
 
-export const transformData = data => ({
-  ...data,
+export const transformData = ({ socialLink, ...rest }) => ({
```

Comment:
Can you also add specializations, addresses and therapies, before spreading it, e.g `socialLink, specializations, addresses, therapies, ...rest`.

Reply:
Sure thing, fixed it!